src/containers/Simularium/index.tsx

```diff
@@ -102,10 +104,26 @@ class App extends React.Component<AppProps, AppState> {
                 clearUrlParams();
                 setSimulariumController(controller);
             }
-        } else if (userTrajectoryUrl) {
+        };
+
+        const loadUserTrajectoryUrl = (
+            userTrajectoryUrl: string | string[]
+        ) => {
             const verifiedUrl = urlCheck(userTrajectoryUrl);
             const fileId = getFileIdFromUrl(verifiedUrl, parsed.id);
-            if (verifiedUrl) {
+            if (
+                verifiedUrl &&
+                fileId &&
+                USER_TRAJ_REDIRECTS.includes(verifiedUrl)
+            ) {
+                // Edge case where we want to redirect to a networked file
+                const streamingUrl =
+                    location.origin +
+                    location.pathname +
+                    `?trajFileName=${fileId}`;
```

Comment:
Nit:  Mixing string concatenations? you could just make this a single formatted string like`${location.origin}${location.pathname}?trajFileName=${fileId}` or use all `+` operations.

Reply:
Also, I wonder if the whole thing could just be moved into that user url handling utils file.

Reply:
Okay, I removed all the `+`s and moved this to utils :) Writing the test for this new util function was tricky because I couldn't figure out how to set `window.location` to whatever I wanted. Let me know if you or @meganrm know a better way to write it.